disco/types/guild.py

```diff
@@ -72,7 +72,7 @@ def delete(self, **kwargs):
 
     @property
     def url(self):
-        return 'https://discordapp.com/api/emojis/{}.{}'.format(self.id, 'gif' if self.animated else 'png')
+        return 'https://cdn.discordapp.com/emojis/{}.{}'.format(self.id, 'gif' if self.animated else 'png')
```

Comment:
Drop this it will get pulled in from your other commit to master when v1 stuff gets merged.

Reply:
Done